Index.html

```diff
@@ -21,54 +21,8 @@
     <link href="styles/Main.css" rel="stylesheet">
     <link rel="manifest" href="site.webmanifest" />
     <meta name="msapplication-TileColor" content="#da532c" />
-    <meta name="theme-color" content="#ffffff" />\
-    <style>
-      h1{ 
```

Comment:
In this lab there *were* a couple places where I did ask you to make your styles inline because they were specific to a particular page. But most of your styles should be in the external stylesheet so that they apply to all pages in which the stylesheet is linked